src/common/csirf.cpp

```diff
@@ -297,6 +297,40 @@ cSIRF_ratio(const void* ptr_x, const void* ptr_y)
 	CATCH;
 }
 
+extern "C"
+void*
+cSIRF_maximum(const void* ptr_x, const void* ptr_y)
+{
+	try {
+		DataContainer& x =
+			objectFromHandle<DataContainer >(ptr_x);
```

Comment:
```suggestion
		auto& x = objectFromHandle<DataContainer >(ptr_x);
```
is less verbose maybe. In fact, at least for some of these you should probably do
```
		auto const& x = objectFromHandle<DataContainer >(ptr_x);
```